test/common/testlib.py

```diff
@@ -1380,12 +1381,18 @@ def check_journal_messages(self, machine=None):
             "SYSLOG_IDENTIFIER=cockpit-ws",
             "SYSLOG_IDENTIFIER=cockpit-bridge",
             "SYSLOG_IDENTIFIER=cockpit/ssh",
+            # also catch GLIB_DOMAIN=<library> which apply to cockpit-ws (but not to -bridge, too much random noise)
+            "_COMM=cockpit-ws",
             "GLIB_DOMAIN=cockpit-ws",
             "GLIB_DOMAIN=cockpit-bridge",
             "GLIB_DOMAIN=cockpit-ssh",
             "GLIB_DOMAIN=cockpit-pcp"
         ]
 
+        # older c-ws versions always log an assertion, fixed in PR #16765
+        if self.image == "rhel-8-5-distropkg":
+            self.allowed_messages.append("json_object_get_string_member: assertion 'node != NULL' failed")
```

Comment:
glad i get to carry this brown paper bag around for a while.

is this the kind of thing that it makes sense to distro-patch?

Reply:
Not for z-stream. Only security and important (data loss etc.) issues there, and for 8.6 it'll be fixed. We can drop this in a few months when we move to 8-6-distropkg.